src/script/components/MessagesList/Message/ContentMessage/asset/FileAssetComponent.test.tsx

```diff
@@ -47,49 +37,45 @@ describe('FileAssetComponent', () => {
     return message;
   }
 
-  it('renders file uploads', () => {
-    const teamState = {
-      isFileSharingReceivingEnabled: ko.pureComputed(() => true),
-    } as TeamState;
+  const teamState = {
+    isFileSharingReceivingEnabled: ko.pureComputed(() => true),
+  } as TeamState;
 
-    const testPage = new FileAssetComponentTestPage({
+  it('renders file uploads', () => {
+    const props = {
       message: mockContentMessage(),
       teamState,
-    });
+    };
 
-    const fileUploadMessage = testPage.getFile();
-    expect(fileUploadMessage).not.toBeNull();
+    const {queryByTestId} = render(<FileAssetComponent {...props} />);
+
+    expect(queryByTestId('file')).not.toBeNull();
   });
 
   it('does not render file uploads from timed-out / obfuscated messages', () => {
-    const teamState = {
-      isFileSharingReceivingEnabled: ko.pureComputed(() => true),
-    } as TeamState;
-
     const message = mockContentMessage();
     message.ephemeral_expires(true);
     message.status(StatusType.SENT);
 
-    const testPage = new FileAssetComponentTestPage({
+    const props = {
       message,
       teamState,
-    });
+    };
+
+    const {queryByTestId} = render(<FileAssetComponent {...props} />);
 
-    const fileUploadMessage = testPage.getFile();
-    expect(fileUploadMessage).toBeNull();
+    const fileElement = queryByTestId('file');
+    expect(fileElement).toBeNull();
```

Comment:
In other cases you passing immediately to expect this queryByTestId, please make it consistent.
```suggestion
    expect(queryByTestId('file')).toBeNull();
```